openmpi.yaml

```diff
@@ -0,0 +1,59 @@
+package:
+  name: openmpi
+  version: 4.1.6
+  epoch: 0
+  description: Message passing library for high-performance computing
+  target-architecture:
+    - x86_64
+  copyright:
+    - license: BSD-3-Clause
+
+environment:
+  contents:
+    packages:
+      - autoconf
+      - automake
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - gfortran
+      - hwloc-dev
+      - libevent-dev
+      - libgomp
+      - perl
+
+pipeline:
+  - uses: fetch
+    with:
+      expected-sha256: f740994485516deb63b5311af122c265179f5328a0d857a567b85db00b11e415
+      uri: https://www.open-mpi.org/software/ompi/v4.1/downloads/openmpi-${{package.version}}.tar.bz2
```

Comment:
can we use a mangled version transform to avoid the hardcoded 4.1 in this URL?

Reply:
i.e. similar to https://github.com/wolfi-dev/os/blob/b7fe3f67b76e0d7cdef73d1ee47a640ce0367546/gnutls.yaml#L13-L18